### PR TITLE
filter by scoped price, close #1042

### LINKED ISSUE #1042: filter by scoped price

### PR DESCRIPTION
**Pull request by schleichardt**

### CODE REVIEW

**Comment by schleichardt** [comment]
@lauraluiz please review

File: commercetools-sdk-base/src/main/java/io/sphere/sdk/client/SphereAuthConfig.java [hunk 1]
```diff
@@ -5,7 +5,8 @@
 /**
  * Contains the configuration to fetch access keys for SPHERE.IO.
  */
-public interface SphereAuthConfig {
+public interface
+SphereAuthConfig {
```
Comment (lauraluiz) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/SphereAuthConfig.java]: Why the enter?

Comment (schleichardt) [inline commercetools-sdk-base/src/main/java/io/sphere/sdk/client/SphereAuthConfig.java]: accidentally happened

File: commercetools-models/src/test/java/io/sphere/sdk/products/search/ScopedPriceSearchIntegrationTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    @BeforeClass
+    public static void createData() {
+        ProductFixtures.deleteProductsAndProductTypes(client());
+        final ProductTypeDraft productTypeDraft =
+                ProductTypeDraft.of(PRODUCT_TYPE_KEY, PRODUCT_TYPE_KEY, PRODUCT_TYPE_KEY, emptyList());
+        productType = client().executeBlocking(ProductTypeCreateCommand.of(productTypeDraft));
+    }
+
+    @Test
+    public void filterByCurrentValueCentAmountAndCountry() {
```
Comment (schleichardt) [inline commercetools-models/src/test/java/io/sphere/sdk/products/search/ScopedPriceSearchIntegrationTest.java]: name is outdated is by value

**Comment by lauraluiz** [comment]
:+1:

File: commercetools-models/src/test/java/io/sphere/sdk/products/search/ScopedPriceSearchIntegrationTest.java (hunk 2 above)
Comment (schleichardt) [inline commercetools-models/src/test/java/io/sphere/sdk/products/search/ScopedPriceSearchIntegrationTest.java]: I will fix it in master before the release.

**Pull request merged**